csrc/scheduler/normalization_utils.cpp

```diff
@@ -1621,6 +1618,49 @@ void schedulePersistentKernel(
       reduction_tvs,
       unroll_vectorizable_cached_tvs);
 
+  // For inner persistent with vectorized load, use explicity unroll for cached
+  // input if it is a persistent buffer. This won't increase register usage
+  // and encourages compiler issuing memory load instructions together. It
+  // improves performance with cuda-13.0.
+  bool unroll_persistent_cached_inputs = rparams->vectorize_inner_reduction &&
+      rparams->fastest_dim && !rparams->schedule_3D;
+  if (unroll_persistent_cached_inputs) {
+    for (auto tv : cached_inputs) {
+      if (std::find(persistent_buffers.begin(), persistent_buffers.end(), tv) ==
+          persistent_buffers.end()) {
+        continue;
+      }
+      // Find PersistentBatch domain to unroll, typical case is:
+      // [..., PersistentBatch, US, TIDx, Vect].
+      // From first principle, the PersistentBatch domain was created from
+      // an outer split and parallelized with Serial, and its content equals
+      // rparams->batches_per_block_inner_reduction, we should have only one
+      // such domain.
```

Comment:
This looks good for now.

I wonder if the index is the same across all persistent buffer, and if we should switch to add `persistent_batch_index` in rparams to avoid the pattern matching here and avoid ambiguity.